templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -930,6 +930,56 @@ export default {
         if (!validPreviousAnswers) {
           errMsgs.push("Please select a previous activity to display")
         }
+
+        const questionsWithMultipleChoiceEnabled = this.node.typeData.activity.questions.filter(
+          question => {
+            return question.answerTypes.multipleChoice.enabled
+          }
+        )
+
+        const validMultipleChoiceValues = questionsWithMultipleChoiceEnabled.every(
```

Comment:
Might be cleaner to throw these types of long functions into a method or computed property